core/src/main/java/org/elasticsearch/index/query/AbstractQueryBuilder.java

```diff
@@ -42,6 +43,8 @@
 
     /** Default for boost to apply to resulting Lucene query. Defaults to 1.0*/
     public static final float DEFAULT_BOOST = 1.0f;
+    public static final ParseField NAME_FIELD = new ParseField("_name");
+    public static final ParseField BOOST_FIELD = new ParseField("boost");
```

Comment:
I think we have to decide whether ParseFields should be in query builders or query parsers. Good to share these two though given that they are always the same. They should indeed be in some base class.